testing-suite/staking-v4/scenarios/PR_6114.py

```diff
@@ -0,0 +1,181 @@
+import threading
+import time
+
+from core.validatorKey import *
+from core.chain_simulator import *
+from chain_commander import *
+from staking import stake, unStake, unBondNodes
+from get_infos.get_validator_info import get_keys_state, get_keys_from_validator_statistics, \
+    get_keys_from_validator_auction
+import requests
+
+
+# SCENARIO 1
+# Have every epoch auction list with enough nodes (let's say 8 qualified, 2 unqualified)
+# Unstake 1 or more eligible/waiting nodes (no more than all nodes per shard or no more than all eligible)
+# and call auction list api; we should see 8 qualified nodes. Next epoch we need to check that exactly 8 nodes were qualified.
+# Afterwards(some epochs), those leaving nodes will be leaving and replaced by the auction nodes
+
+
+# Steps:
+#
+# 1) In epoch 5 stake with A that 2 keys  - will go in auction
+# 2) Add blocks and epochs until 1 of this 2 keys are eligible - let's call it epoch X
+# 3) unStake the eligible key in epoch X
+# 4) call auction list api after 2 blocks (we are still in epoch X) and:
+#       4.1) unStaked key status should be "eligible (leaving)" on validator/statistics
+#       4.2) we should have still 8 keys qualified
+#       4.3) we should have still 2 keys not qualified
+#       4.4) we should have 39 keys with status "eligible"
+#       4.5) we should have still 16 keys with status "waiting"
+# 5) go to epoch X+1  and 5 blocks and make sure:
+#       5.1) unstaked key should be inactive
+#       5.2) 8 keys from auction (qualified or not qualified , cause is not deterministic)  are now waiting
+#       5.3) we still have 9 keys in auction, 8 of them are qualified, 1 not qualified , 40 keys eligible, 16 waiting
+
+
+def main():
+    print("Happy testing")
+
+
+def test_PR_6114():
+
+    # === PRE-CONDITIONS ==============================================================
+    AMOUNT_TO_MINT = "10000" + "000000000000000000"
```

Comment:
Better just have a func to create "egld" value from your amount, instead of this

Reply:
No. It is needed to be store in a variable to be checked later if the minted was done.

Reply:
It will still be a variable, but having a func, like:
```
 AMOUNT_TO_MINT =  getEGLDAmount("10000")
```

which adds the remaining necessary num of decimals. It should also take care of values < 1 egld

Reply:
Will be added only if we have a task defined in JIRA for that, otherwise, it will be ignored and forgotten